sphinx/config.py

```diff
@@ -78,7 +76,7 @@ def match(self, value):
             return value in self.candidates
 
 
-string_classes = [text_type]  # type: List
+string_classes = [str]  # type: List
```

Comment:
Originally, `string_classes` has been used to switch string classes with python interpreters:
https://github.com/sphinx-doc/sphinx/blob/c899b27b4a659f95a20d80bf467f1e6d21ffb883/sphinx/config.py#L85-L87

After our refactoring, it does not have any role. So it would be better to remove this now.

Reply:
Done in #5818.